finagle-http/src/main/scala/com/twitter/finagle/http/filter/StatsFilter.scala

```diff
@@ -23,6 +23,12 @@ class StatsFilter[REQUEST <: Request](stats: StatsReceiver)
   private[this] val timeReceiver = stats.scope("time")
   private[this] val responseSizeStat = stats.stat("response_size")
 
+  private[this] val counterCache: Map[String, Counter] =
+    Map.empty.withDefault(statusReceiver.counter(_))
```

Comment:
I tried using `Memoize` from twitter-util, which works. I've verified that by seeing the effect of println only once. However, I'm having issues verifying from the outside of `StatsFilter` that it actually works. `counter()` is being called twice for two requests.